src/parse/matches/arg_matches.rs

```diff
@@ -466,7 +474,7 @@ impl ArgMatches {
     pub fn values_of_t_or_exit<R>(&self, name: &str) -> Vec<R>
     where
         R: FromStr,
-        <R as FromStr>::Err: Display,
+        <R as FromStr>::Err: Display + Debug,
```

Comment:
Debug

Reply:
Because of forwarding to `values_of_t` and because `values_of_t` has `parsed.expect`